src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,199 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    public static class Node<T> {
+        private T data;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T element, Node<T> next) {
+            this.data = element;
+            this.prev = prev;
+            this.next = next;
+        }
+
+        public T getData() {
+            return data;
+        }
```

Comment:
[checklist issue](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#dont-use-getters-and-setters-in-class-node)